proposals/nnnn-cloud-app-transport-adapter.md

```diff
@@ -0,0 +1,243 @@
+# Cloud App Transport Adapter
+
+* Proposal: [SDL-NNNN](NNNN-cloud-app-transport-adapter.md)
+* Author: [Jack Byrne](https://github.com/smartdevicelink)
+* Status: **Awaiting review**
+* Impacted Platforms: [Core]
+
+## Introduction
+
+This proposal will detail a possible solution for allowing  SDL-enabled cloud applications to be used in a vehicle. This proposal will cover the process of obtaining web application server endpoints, opening connections between Core and a cloud based app server, and how a web application can authenticate connected head units. 
+
+The implementation of a SDL web app javascript library used to create SDL connected web applications will be covered by another proposal. 
+
+
+## Motivation
+
+### Background on SDL-enabled Web Applications:
+- SDL web apps would still use the SDL RPC Service / protocol
+- SDL web apps would integrate a proxy library that supports all Mobile API RPCs (implemented in Javascript)
+- SDL web apps live in the "cloud", not in the car or phone
+- Using a web app on the head unit would not require mobile a mobile device, but does require head unit to have an active data connection
```

Comment:
Please update to "...would not require a mobile device, but would require the head unit to have..."